### DISCUSSION TITLE
3.02 use of comments instead of docstring

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/2nd-semester-introduction-to-computer-science#346] [Ideas]
Python has something called a docstring to add documentation to a function in a standardized way.
3.02 shows this:
```
# Name: multiply_by_two
# Purpose: multiplies a given number by 2
# Input: number to multiply (int)
# Output: int
def multiply_by_two(a): 
  c = a * 2
  return c
```
Instead, I would recommend this:
```
def multiply_by_two(a): 
  """
  Name: multiply_by_two
  Purpose: multiplies a given number by 2
  Input: number to multiply (int)
  Returns: int
  """
  c = a * 2
  return c
```
Many IDEs will know about docstrings and display them as you write code.

**Comment by RandyCHS** [comment]
I like this idea of teaching docstrings as well.  This is standard practice now in most 1st year CS courses in college and would require little change in the curriculum.
